e2e/tests/test.ts

```diff
@@ -53,6 +56,26 @@ export function before(
   setupTasks.push(task);
 }
 
+export async function beforeRenderTest(
+  cb: (
+    canvas: Canvas
+  ) => AsyncReturnValue<{ title: string; boundFunction: () => void }[]>,
+  options
+) {
+  const el = document.querySelector<HTMLCanvasElement>('#canvas');
+  const canvas = new Canvas(el, options);
+  // cb has to bind the rendering test to the specific canvas and add a clear before the test
+  const renderingTests = await cb(canvas);
+  renderingTests.forEach((renderTest) => {
+    if (renderingTestMap.has(renderTest.title)) {
+      throw new Error(
+        `test identifiers must be unique: ${renderTest.title} is already defined`
+      );
+    }
+    renderingTestMap.set(renderTest.title, renderTest.boundFunction);
+  });
+}
+
```

Comment:
I am not sure I under this.
Do you want to create a new canvas for every test?
Why? What is the gain?
What about simply resizing the canvas before the test and adding a `canvas.clear()` after the snapshot call?
Seems more straight forward, simpler and faster and can be exposed on the CanvasUtil.